code/modules/antagonists/changeling/changeling.dm

```diff
@@ -468,7 +468,7 @@
 	var/datum/objective/protect/new_objective = new /datum/objective/protect
 	new_objective.owner = owner
 	new_objective.target = master
-	new_objective.explanation_text = "Protect [master.current.real_name], the horrorform."
+	new_objective.explanation_text = "Protect the horrorform." //[master.current.real_name]
```

Comment:
why would children/followers of something call it a horrorform. should say protect your mother changeling or somethin